exercises/matrix/canonical-data.json

```diff
@@ -0,0 +1,79 @@
+{
+  "exercise": "matrix",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "extract row from one number matrix",
+      "property": "row",
+      "input": {
+        "string": "1",
+        "index": 0
+      },
+      "expected": [1]
+    },
+    {
+      "description": "can extract row",
+      "property": "row",
+      "input": {
+        "string": "1 2\n3 4",
+        "index": 1
+      },
+      "expected": [3,4]
+    },
+    {
+      "description": "extract row where numbers have different widths",
+      "property": "row",
+      "input": {
+        "string": "1 2\n10 20",
+        "index": 1
+      },
+      "expected": [10,20]
```

Comment:
this line is missing the space after its comma (and the space is present in other lines)